<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="coffee_table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>730</width>
      <height>250</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>type</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>degree</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ground_or_whole</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>taste</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>price</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>volume</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="add_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="change_button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
